website/events/models/event_registration.py

```diff
@@ -63,7 +64,8 @@ def is_registered(self):
     def queue_position(self):
         if self.event.max_participants is not None:
             try:
-                return list(self.event.queue).index(self) + 1
+                queue_ids = [r.member_id for r in self.event.queue]
```

Comment:
What is `r`?

Reply:
Oh and I kinda fail to see the usefulness of this change :p

Reply:
Before this change, looking up the queue position required looking up the member. This happens because I think str is used, according to the stacktrace. And this requires fetching the member's full name. Just using their id's prevents this, since these are stored inside the registration. (I do assume the member_ids are unique in the queue, which always is true since cancelled registrations are not in there).